### PR TITLE
Feature/jc19 442 display detail

### PR DESCRIPTION
**Pull request by LocTran-saritasa**

### CODE REVIEW

File: apps/angular/src/shared/pipe/pluck.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  name: 'pluck',
+})
+export class PluckPipe implements PipeTransform {
+  /**
+   * Transform array to string.
+   * @param input Input array.
+   * @param key Key value.
+   */
+  public transform<T>(input: readonly T[], key: string): string {
+    return input.map(value => value[key as keyof T]).join(', ');
```
Comment (panov-aleksey-saritasa) [inline apps/angular/src/shared/pipe/pluck.pipe.ts:14]:
```suggestion
  public transform<T extends object>(input: readonly T[], key: keyof T): string {
    return input.map(value => value[key]).join(', ');
```

File: apps/angular/src/shared/pipe/safe.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export class SafePipe implements PipeTransform {
+
+  public constructor(private sanitizer: DomSanitizer) { }
+
+  /**
+   * Transform to safe url.
+   * @param url Unsafe url.
+   */
+  public transform(url: string): SafeResourceUrl {
+    return this.sanitizer.bypassSecurityTrustResourceUrl(url);
```
Comment (panov-aleksey-saritasa) [inline apps/angular/src/shared/pipe/safe.pipe.ts:17]: Shouldn't you use [sanitize](https://angular.io/api/platform-browser/DomSanitizer#sanitize) method instead?

File: apps/angular/src/shared/pipe/safe.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+
+/** Safe url for iframe tag. */
+@Pipe({
+  name: 'safe',
+})
+export class SafePipe implements PipeTransform {
+
+  public constructor(private sanitizer: DomSanitizer) { }
```
Comment (panov-aleksey-saritasa) [inline apps/angular/src/shared/pipe/safe.pipe.ts:10]:
```suggestion
  public constructor(private readonly sanitizer: DomSanitizer) { }
```

File: apps/angular/src/shared/pipe/safe.pipe.ts (hunk 2 above)
Comment (LocTran-saritasa) [reply apps/angular/src/shared/pipe/safe.pipe.ts:17]: `return this.sanitizer.sanitize(SecurityContext.RESOURCE_URL, url);`
I tried this but it not work for me.

**Pull request closed without merging**